ci/codestyle.sh

```diff
@@ -15,12 +15,11 @@ done
 echo "ok"
 fi
 
-# Will uncomment this once we reformat
-#if command -v clang-format; then
-#    echo -n "checking clang-format... "
-#    git ls-files '**.c' '**.cxx' '**.h' '**.hpp' | xargs clang-format --Werror --dry-run
-#    echo "ok"
-#fi
+if command -v clang-format; then
+    echo -n "checking clang-format... "
+    git ls-files '**.c' '**.cxx' '**.h' '**.hpp' | xargs clang-format --Werror --dry-run
```

Comment:
Having to copy-paste this invocation and modify it to apply `clang-format` the first time before it goes in your shell history isn't ideal. Let's add a `clang-format` make target for this? Like in https://github.com/coreos/rpm-ostree/commit/14a8b7c92235da56acb977e9ba5029006c3acb0b.

Reply:
> let's add a clang-format make target for this? Like in https://github.com/coreos/rpm-ostree/commit/14a8b7c92235da56acb977e9ba5029006c3acb0b.

That already landed in https://github.com/ostreedev/ostree/blob/main/Makefile.am#L136